### PR TITLE
Missing signed commit display translations

### PR DESCRIPTION
**Pull request by lafriks**
Untranslated strings when displaying signed commit info

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -581,6 +581,8 @@ commits.message = Message
 commits.date = Date
 commits.older = Older
 commits.newer = Newer
+commits.signed_by = Signed by
+commits.gpg_key_id = GPG key ID
```
Comment (bkcsoft) [inline options/locale/locale_en-US.ini:585]: I don't see this one used in this PR, was it missing?

Comment (lafriks) [reply options/locale/locale_en-US.ini:585]: @bkcsoft fixed, I was testing if it works correctly and doing undo/redo and forgot to save latest code changes in editor :)

**Comment by bkcsoft** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

**Pull request merged**